modules/social_features/social_group/modules/social_group_invite/src/Form/SocialBulkGroupInvitation.php

```diff
@@ -247,8 +247,8 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $invite_settings = $this->configFactory->get('social_group.settings')->get('group_invite');
 
     // Set preview subject and message.
-    $invitation_subject = $invite_settings['invite_subject'] ?? $group_invite_config['invitation_subject'];
-    $invitation_body = $invite_settings['invite_message'] ?? $group_invite_config['invitation_body'];
+    $invitation_subject = $group_invite_config['invitation_subject'] ?? $invite_settings['invite_subject'];
```

Comment:
Hi @nechai,
What will happen for sites that have customised invitation text?
It looks for me that it will be set to default text after this change, which is not very good, so question is iff we can have smooth  upgrade path in such case?